omod/src/main/java/org/openmrs/module/fhir/providers/RestfulEncounterResourceProvider.java

```diff
@@ -68,6 +71,18 @@ public Encounter getResourceById(@IdParam IdDt theId) {
 		return encounterResource.searchEncountersById(id);
 	}
 
+	/**
+	 * Search encounters by patient identifier
+	 *
+	 * @param identifier object containing the patient identifier
+	 */
+	@Search()
+	public List<Encounter> searchEncountersByPatientIdentifier(
+		@RequiredParam(name=Encounter.SP_PATIENT, chainWhitelist= {Patient.SP_IDENTIFIER}) ReferenceParam identifier
```

Comment:
I see a slight formatting issue in here. Can you correct that?
Also if you search by patient identifier can't it be like https://github.com/milankarunarathne/openmrs-module-fhir/blob/FM-15/omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPractitionerResourceProvider.java#L76 why  chainlist need to be involve in here?